src/toLowerCase.test.ts

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-useless-escape */
```

Comment:
put this above the line that needs it. If you need it for a block prefer wrapping that area with:

```ts
/* eslint-disable no-useless-escape */
...
...
...
/* eslint-enable no-useless-escape */
```